ScreenToGif/Resources/Localization/StringResources.el.xaml

```diff
@@ -59,553 +78,623 @@
-    <s:String x:Key="S.Updater.RunAfter">Εκτελέστε την εφαρμογή μετά την ενημέρωση.</s:String>
+    <s:String x:Key="S.Updater.Portable">Portable</s:String>
+    <s:String x:Key="S.Updater.Installer">Installer</s:String>
+    <s:String x:Key="S.Updater.Info.WhatsNew">Τι νέο υπάρχει;</s:String>
+    <s:String x:Key="S.Updater.Info.BugFixes">Bug fixes:</s:String>
+    <s:String x:Key="S.Updater.Info.NewVersionAvailable">A new version is available for you to download.&#x0d;Would you like to open the browser to download it?</s:String>
+    <s:String x:Key="S.Updater.RunAfter">Εκτέλεση της εφαρμογής ματά την ενημέρωση.</s:String>
     <s:String x:Key="S.Updater.Download">Λήψη</s:String>
     <s:String x:Key="S.Updater.Install">Εγκατάσταση</s:String>
-    <s:String x:Key="S.Updater.InstallManually">Εγκαταστήστε με μη αυτόματο τρόπο</s:String>
-    <s:String x:Key="S.Updater.Downloading">Λήψη ...</s:String>
-    <s:String x:Key="S.Updater.Warning.Show">Δεν ήταν δυνατή η εμφάνιση των λεπτομερειών λήψης.</s:String>
+    <s:String x:Key="S.Updater.InstallManually">Install manually</s:String>
+    <s:String x:Key="S.Updater.Downloading">Λήψη...</s:String>
+    <s:String x:Key="S.Updater.Warning.Show">It was not possible to show the download details.</s:String>
     <s:String x:Key="S.Updater.Warning.Download">Δεν ήταν δυνατή η λήψη της ενημέρωσης.</s:String>
-    <s:String x:Key="S.Updater.Warning.Encoding">Πρέπει να περιμένετε να ολοκληρωθούν όλες οι κωδικοποιήσεις πριν από την ενημέρωση.</s:String>
+    <s:String x:Key="S.Updater.Warning.Encoding">You need to wait for all encodings to finish before updating.</s:String>
     
     <!--Downloader-->
     <s:String x:Key="S.Downloader.Title">ScreenToGif - Λήψη</s:String>
-    <s:String x:Key="S.Downloader.Header">Πρόγραμμα λήψης</s:String>
+    <s:String x:Key="S.Downloader.Header">Downloader</s:String>
     <s:String x:Key="S.Downloader.Size">{0} από {1}</s:String>
     
     <!--Options-->
-    <s:String x:Key="S.Options.Title">ScreenToGif - Ρυθμίσεις</s:String>
+    <s:String x:Key="S.Options.Title">ScreenToGif - Επιλογές</s:String>
     <s:String x:Key="S.Options.App">Εφαρμογή</s:String>
-    <s:String x:Key="S.Options.Recorder">Καταγραφέας</s:String>
-    <s:String x:Key="S.Options.Editor">Πρόγραμμα Επεξεργασίας</s:String>
+    <s:String x:Key="S.Options.Recorder">Recorder</s:String>
+    <s:String x:Key="S.Options.Editor">Επεξεργαστής</s:String>
     <s:String x:Key="S.Options.Tasks">Εργασίες</s:String>
     <s:String x:Key="S.Options.Shortcuts">Συντομεύσεις</s:String>
     <s:String x:Key="S.Options.Language">Γλώσσα</s:String>
-    <s:String x:Key="S.Options.Storage">Αποθηκευτικός χώρος</s:String>
-    <s:String x:Key="S.Options.Extras">Πρόσθετα</s:String>
-    <s:String x:Key="S.Options.Upload">Νέφος</s:String>
+    <s:String x:Key="S.Options.Storage">Storage</s:String>
```

Comment:
Sorry for the delay in checking this out, but I see that some translations were undone (this one for example, but there's several others).